bots/util.py

```diff
@@ -2236,36 +2263,33 @@ def _monitor_method(self):
             self.wait_for_next_cycle()
             self.iterate_display_index()
 
-            bean_wsteth_liquidity = 0
-            bean_wsteth_volume = 0
-            bean_eth_liquidity = 0
-            bean_eth_volume = 0
+            liquidity = 0
+            volume = 0
+            well_count = 0
             wells = self.basin_graph_client.get_wells_stats()
 
             for well in wells:
-                if well["id"].lower() == BEAN_ETH_WELL_ADDR.lower():
-                    bean_eth_liquidity += float(well["totalLiquidityUSD"])
-                    bean_eth_volume += float(well["cumulativeTradeVolumeUSD"])
-                elif well["id"].lower() == BEAN_WSTETH_WELL_ADDR.lower():
-                    bean_wsteth_liquidity += float(well["totalLiquidityUSD"])
-                    bean_wsteth_volume += float(well["cumulativeTradeVolumeUSD"])
-
-            if bean_eth_liquidity == 0 or bean_wsteth_liquidity == 0:
+                if well["id"].lower() in [BEAN_ETH_WELL_ADDR.lower(), BEAN_WSTETH_WELL_ADDR.lower()]:
+                    liquidity += float(well["totalLiquidityUSD"])
+                    volume += float(well["cumulativeTradeVolumeUSD"])
+                    well_count += 1
```

Comment:
this may cause rate limiting when number of wells gets large, but probably not an issue yet